### DISCUSSION TITLE
Is a mobile-friendly version (either android, ios, or PWA) in the roadmap?

### DISCUSSION THREAD

**Discussion by mdayaram** [bangle-io/bangle-io#414] [General]
Hello! I came across bangle from a blog post.  I've been bouncing around different note taking apps for a while now, and bangle seems like it would be great in terms of the functionality I need as well as the values I look for. However, I was having trouble using the current web page on mobile, which is essential for my workflows.  I tried searching a bit and I couldn't find any mobile-specific version of the app (nothing in github issues nor this discussion forum), so figured I'd ask if it's in the roadmap to add that sort of support, or perhaps it's not a focus for the project?

In case it's interesting or if anyone is curious, my normal workflow tends to be "generate a lot of notes on mobile that go into an inbox folder" while I go through my day, and occasionally "search notes via tags or keywords on mobile" when I'm about to go into an event that I could use my notes context on.  Then every so often (a few times a week) I use my desktop to organize/edit/add metadata/cleanup my notes.

**Comment by kepta** [comment]
Hey @mdayaram thanks for opening this thread. Yes Bangle.io should work on mobile devices as a web app. I have tested it out on iOS devices and it works well for me. I donot have android device, so I am not sure how well it works in that.

If you want to try it out, I would recommend choosing Github storage when creating a workspace so that you can sync between multiple devices.

**Comment by mdayaram** [comment]
I see, yes, I can get around okay, but the interface is a bit clunky since it relies a lot on keyboard shortcuts which are very awkward to use in mobile.  I still have to browse to the page though (where as usually a PWA will ask me to install it on my phone as an app badge).  

So rephrasing my question into two:
* Is a mobile-friendly navigation and interaction interface in the roadmap at all? 
* Is there a way to install bangle as a PWA app on my phone that perhaps I'm not aware of? (usually a PWA app will prompt to be installed when I browse to it, e.g. https://zeitgeistsf.square.site/

**Reply by kepta** [reply]
>  but the interface is a bit clunky since it relies a lot on keyboard shortcuts

Yes I agree it is very clunky right now and I have plans to address them. If you are interested in helping out, it will be great if you could create an issue with some ideas on what can be done to improve the UX. Some examples from other mobile websites for inspiration will be great.

> Is a mobile-friendly navigation and interaction interface in the roadmap at all?

Yes this is something I am actively working on fixing. Sadly, I haven't gotten the time to cleanup the roadmap and make it public. I will update this thread when I have a public facing roadmap.

> Is there a way to install bangle as a PWA app on my phone that perhaps I'm not aware of? 

If you are on iOS and click on the share button (it is the middle one), you will see an option `Add to Home Screen`. Clicking it will install it as a PWA. Let me know if you face any hurdles.

**Comment by mdayaram** [comment]
> If you are on iOS and click on the share button (it is the middle one), you will see an option Add to Home Screen. Clicking it will install it as a PWA. Let me know if you face any hurdles.

I'm on Android, but by following similar instructions, I was able to install the app! Thank you!

> Yes this is something I am actively working on fixing. Sadly, I haven't gotten the time to cleanup the roadmap and make it public. I will update this thread when I have a public facing roadmap.

Awesome! Looking forward to it! I'm very excited for bangle's potential given your current values and direction!
